resources/icons.css

```diff
@@ -398,6 +398,12 @@
     background-position: -985px 0px !important;
 }
 
+.icon-line-solid {
+    width: 28px;
+    height: 28px;
+    background-color: black !important;
+}
+
```

Comment:
resources/icons.css is generated with a script. Any manual changes here will be overwritten next time it's run. We should put these styles elsewhere and give them some other name so they are not confused with regular image icons.